src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudFoundryApplication.java

```diff
@@ -43,9 +43,10 @@
 @ImplementedBy(VanillaCloudFoundryApplicationImpl.class)
 public interface VanillaCloudFoundryApplication extends Entity, Startable, DriverDependentEntity {
 
-    @SetFromFlag("name")
-    ConfigKey<String> APPLICATION_NAME = ConfigKeys.newStringConfigKey(
-            "cloudFoundry.application.name", "Name of the application");
+    @SetFromFlag("name-app")
```

Comment:
I found this flag, https://github.com/apache/brooklyn-server/blob/master/core/src/main/java/org/apache/brooklyn/core/entity/AbstractEntity.java#L376

We could reuse it instead of this `ConfigKey`?